tests/func/test_api.py

```diff
@@ -69,20 +74,31 @@ def test_open(tmp_dir, dvc, remote):
 @pytest.mark.parametrize(
     "cloud",
     [
-        pytest.lazy_fixture(cloud)
-        for cloud in [
-            "real_s3",  # NOTE: moto's s3 fails in some tests
-            "gs",
-            "azure",
-            "gdrive",
-            "oss",
-            "ssh",
-            "hdfs",
-            "http",
-        ]
+        *[
+            pytest.lazy_fixture(cloud)
+            for cloud in [
+                "real_s3",  # NOTE: moto's s3 fails in some tests
+                "gs",
+                "azure",
+                "gdrive",
+                "oss",
+                "hdfs",
+                "http",
+            ]
+        ],
+        pytest.param(
+            pytest.lazy_fixture("ssh"),
+            marks=pytest.mark.xfail(
+                raises=FileNotFoundError, condition="sys.platform == 'win32'"
+            ),
+        ),
```

Comment:
This happens only in this PR, but there is no sign of that in comments/reasons.

Reply:
Is there a master ticket with all the things that need to be fixed?

Reply:
I will include the list in original issue